build.sbt

```diff
@@ -189,4 +197,8 @@ lazy val data = project
     reStart / javaOptions ++= Seq("-Xmx4g")
   )
   .enablePlugins(BuildInfoPlugin, JavaAppPackaging)
-  .dependsOn(model, search)
+  .dependsOn(core, infra)
+
+lazy val V = new {
+  val doobieVersion = "0.13.4"
```

Comment:
Would be better to group all the versions together. So I suggest to move the old ones here. https://github.com/scalacenter/scaladex/blob/9dc8ca0f6f92bf6c75625724c3ac660ebfa9558d/build.sbt#L6-L11